camel/models/anthropic_model.py

```diff
@@ -48,17 +49,19 @@ class AnthropicModel(BaseModelBackend):
     def __init__(
         self,
         model_type: ModelType,
-        model_config_dict: Dict[str, Any],
+        model_config_dict: Optional[Dict[str, Any]] = None,
         api_key: Optional[str] = None,
         url: Optional[str] = None,
         token_counter: Optional[BaseTokenCounter] = None,
     ) -> None:
+        if not model_config_dict:
+            model_config_dict = AnthropicConfig().as_dict()
```

Comment:
Should we also overwrite the config if the user explicitly passed in an empty config dict?